[2024-03-03T03:41:39.260+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-03T00:00:00+00:00 [queued]>
[2024-03-03T03:41:39.287+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-03T00:00:00+00:00 [queued]>
[2024-03-03T03:41:39.289+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:41:39.327+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_3> on 2016-12-03 00:00:00+00:00
[2024-03-03T03:41:39.339+0000] {standard_task_runner.py:60} INFO - Started process 4503 to run task
[2024-03-03T03:41:39.344+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_3', 'scheduled__2016-12-03T00:00:00+00:00', '--job-id', '2050', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmp9fr0kp2j']
[2024-03-03T03:41:39.349+0000] {standard_task_runner.py:88} INFO - Job 2050: Subtask get_files_3
[2024-03-03T03:41:39.376+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:41:39.439+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_3 scheduled__2016-12-03T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:41:39.556+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_3' AIRFLOW_CTX_EXECUTION_DATE='2016-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-12-03T00:00:00+00:00'
[2024-03-03T03:41:41.897+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:41:41.914+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_3, execution_date=20161203T000000, start_date=20240303T034139, end_date=20240303T034141
[2024-03-03T03:41:41.971+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:41:42.008+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
